code/modules/projectiles/pins.dm

```diff
@@ -171,7 +172,7 @@
 			to_chat(user, "<span class='notice'>DNA-LOCK SET.</span>")
 
 /obj/item/firing_pin/dna/pin_auth(mob/living/carbon/user)
-	if(unique_enzymes && user?.dna?.unique_enzymes == unique_enzymes) //First check is in case they're both null
+	if(unique_enzymes && ((obj_flags & EMAGGED) || user?.dna?.unique_enzymes == unique_enzymes)) //First check is in case they're both null
```

Comment:
If it was emagged but didn't have unique_enzymes that would still return FALSE? Why?

Reply:
To simplify things. Pin auth failure with null `unique_enzymes` sets the var.